Sources/SentrySwiftUI/SentryInternal/SentryInternal.h

```diff
@@ -56,4 +71,46 @@ typedef NS_ENUM(NSUInteger, SentrySpanStatus);
 
 @end
 
+@interface SentryTimeToDisplayTracker : NSObject
```

Comment:
`l:` This header is growing to a point where you should consider moving the class interfaces into own header files which are imported and exported, turning this into an umbrella header file

Reply:
Let's do it in the next iteration, in case we need to add more things in here. 😉